src/azure-cli/azure/cli/command_modules/acs/tests/latest/recordings/test_aks_create_and_update_with_http_proxy_config.yaml

```diff
@@ -329,7 +329,7 @@ interactions:
       - AZURECLI/2.42.0 (PIP) azsdk-python-azure-mgmt-network/21.0.1 Python/3.8.9
         (Windows-10-10.0.22621-SP0)
     method: PUT
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest000001/providers/Microsoft.Network/virtualNetworks/cliakstest000002?api-version=2022-01-01
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest000001/providers/Microsoft.Network/virtualNetworks/cliakstest000002/subnets/proxy-subnet?api-version=2022-01-01
```

Comment:
I'm curious why `/subnets/proxy-subnet` was added to the `uri`.

Reply:
Cauz `az network vnet subnet` used VNet API before (kind of subcommand implementation), and it uses its own independent API now.